src/main/java/pl/exsio/querydsl/entityql/QFactory.java

```diff
@@ -1,7 +1,11 @@
 package pl.exsio.querydsl.entityql;
 
+import com.querydsl.core.Tuple;
 import pl.exsio.querydsl.entityql.entity.metadata.QEntityMetadata;
 import pl.exsio.querydsl.entityql.entity.scanner.QEntityScanner;
+import pl.exsio.querydsl.entityql.entity.scanner.QObjectScanner;
+import pl.exsio.querydsl.entityql.entity.scanner.TableScanner;
+import pl.exsio.querydsl.entityql.entity.scanner.nativesql.Table;
```

Comment:
I would change the package name from "nativesql" to "dynamic". Everything we do here revolves around Native SQL afterall :)

Reply:
Couldn't think of anything original)
In progress....

Reply:
correction: "nativesql" -> "runtime"

Reply:
renamed